src/Components/CaseGrid/CaseThumb.js

```diff
@@ -62,15 +62,9 @@ const Desc = styled.div`
 `
 
 const ProjectName = styled.div`
-  position: absolute;
-  bottom: 0;
-  transform: rotate(-90deg);
-  transform-origin: 0% 0%;
-  left: 10px;
-  font-size: 13px;
-
-  @media ${props => props.theme.media.xs} {
-    width: 50%;
+  font-size: 14px;
+  @media ${props => props.theme.media.lg} {
+    padding-top: 20px;
```

Comment:
Please avoid using magic numbers, as we have a spacing tool, that is making sure that we always use a coherrent spacing: `props => props.theme.spacing(4)`-> this will result in 20px